### DISCUSSION TITLE
add support for flux fooocus simplesdxl2

### DISCUSSION THREAD

**Discussion by finkbom** [LykosAI/StabilityMatrix#855] [Ideas]
https://huggingface.co/metercai/SimpleSDXL2

be great to add this fork of fooocus that is being actively updated

**Comment by steppres** [comment]
I just wanted to point out that incorporating SimpleSDXL2 into StabilityMatrix would be incredibly challenging. The software has numerous unconventional installation methods, and each update tends to introduce new, odd variations. It’s also quite fragile and prone to breaking. Maintaining it would be a massive effort and an ongoing headache to keep updated.
